plugins/modules/vmware_object_role_permission_info.py

```diff
@@ -51,18 +57,28 @@
 """
 
 EXAMPLES = r"""
-- name: Gather permission information about Datastore
+- name: Gather role information about Datastore
+  community.vmware.vmware_object_role_permission_info:
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: false
+    object_name: ds_200
+    object_type: Datastore
+
+- name: Gather permissions on Datastore for a User
   community.vmware.vmware_object_role_permission_info:
     hostname: "{{ vcenter_hostname }}"
     username: "{{ vcenter_username }}"
     password: "{{ vcenter_password }}"
     validate_certs: false
+    principal: some.user@company.com
     object_name: ds_200
     object_type: Datastore
 """
 
 RETURN = r"""
-permission_info:
+role_info:
```

Comment:
Module returns `permission_info` so I would say keep this value as-is. If you want to change the name of the return value then add a separate key. This is to make sure that we have backward compatability. Also, you need to make change code changes in `exit_json` in the given module code.

Reply:
I had thought about changing the key to reflect role vs permission, but ended up leaving the code unchanged for backwards compatibility and forgot to revert the doc.

Reply:
OK.